### PR TITLE
tests: fix incompatibility with pytest>=2.8.0

### PR DESCRIPTION
**Pull request by SamiHiltunen**
- FIX Removes calls to PluginManager
  consider_setuptools_entrypoints() removed in PyTest 2.8.0.

Signed-off-by: Sami Hiltunen sami.mikael.hiltunen@cern.ch

### CODE REVIEW

**Comment by jirikuncar** [comment]
- [x] amend commit messages

**Comment by SamiHiltunen** [comment]
@jirikuncar commit messages amended

File: setup.py [hunk 1]
```diff
@@ -37,13 +37,15 @@
     'Flask>=0.10.1',
     'six>=1.7.2',
     'invenio-base>=0.2.1',
+    'invenio-ext>=0.1.0',
```
Comment (jirikuncar) [inline setup.py]: `0.2.1`?

File: setup.py [hunk 2]
```diff
@@ -37,13 +37,15 @@
...
+    'invenio-utils>=0.1.0',
```
Comment (jirikuncar) [inline setup.py]: `0.1.1`?

File: .travis.yml [hunk 3]
```diff
@@ -46,7 +46,7 @@ before_install:
   - "sudo apt-get update"
   - "sudo apt-get install python-dev"
   - "travis_retry pip install --upgrade pip"
-  - "travis_retry pip install mock"
+  - "travis_retry pip install check-manifest mock"
```
Comment (jirikuncar) [inline .travis.yml]: `coveralls` is missing

**Pull request merged**